### PR TITLE
ResNet-50 || ImageNet Training || Tensorflow 1.8 || Google

### PR DESCRIPTION
**Pull request by sb2nov**
ResNet-50 trained with batch size 1024 on a cloud TPUv2. This is the next iteration of #10 with TF 1.8rc1

- The framework is TensorFlow 1.8.0-rc1, which has not yet been released as of the creation time of this pull request. This RC1 release will include a small change over RC0 which improves checkpointing behavior.

CC'ing a few other members of the team: @frankchn @chrisying @jhseu @saeta

Please let me know if you have any questions!

### CODE REVIEW

**Comment by sb2nov** [comment]
R: @codyaustun

File: ImageNet/train/google_resnet50_tpu_tensorflow18.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "schedule": [
+      {"learning_rate": 0.0, "epochs": 1},
+      {"learning_rate": 0.08, "epochs": 1},
+      {"learning_rate": 0.16, "epochs": 1},
+      {"learning_rate": 0.24, "epochs": 1},
+      {"learning_rate": 0.32, "epochs": 1},
+      {"learning_rate": 0.4, "epochs": 25},
+      {"learning_rate": 0.04, "epochs": 30},
+      {"learning_rate": 0.004, "epochs": 20},
+      {"learning_rate": 0.0004, "epochs": 15}
```
Comment (codyaustun) [inline ImageNet/train/google_resnet50_tpu_tensorflow18.json]: This doesn't add up to the number of epochs you have in the TSV.

**Review by codyaustun — CHANGES_REQUESTED** [review]

File: ImageNet/train/google_resnet50_tpu_tensorflow18.json (hunk 1 above)
Comment (sb2nov) [reply ImageNet/train/google_resnet50_tpu_tensorflow18.json]: Thanks for catching. Fixed it.

**Review by codyaustun — APPROVED** [review]

**Comment by codyaustun** [comment]
Thanks! The website is updated.

**Comment by sb2nov** [comment]
Thanks for the speedy review.

**Pull request merged**